### DISCUSSION TITLE
Downloads randomly pausing?

### DISCUSSION THREAD

**Discussion by edrock200** [nzbgetcom/nzbget#316] [Q&A]
I have random downloads pause in queue. It's not a global pause, e.g. like if disk space is low, it's on specific download entries. The odd part is, if I tick those entries and click resume, they complete successfully. Anyone else run into this and/or know what's causing it? Looking at the log for the item doesn't seem to yield any clues. I caught this one pause while it was downloading at the 11:20:50 mark in the log attached and clicked resume. Only issue I see is the warning about the par file, but again it finished and completed successfully upon resume. 
[example.item.txt](https://github.com/user-attachments/files/16285426/example.item.txt)

Thanks in advance for any tips/tricks.

**Comment by luckedea** [comment]
@edrock200 Does this happen on specific nzbs or just randomly in general?

**Reply by edrock200** [reply]
Just in general. I'd say out of every ~100 1 or 2 will pause.

**Reply by luckedea** [reply]
> Just in general. I'd say out of every ~100 1 or 2 will pause.

This could be extension-related, not necessarily visible in logs.
It would help if you could repeat it without extensions.

Too many things can be happening here, this one is difficult because I can't replicate it.

**Reply by edrock200** [reply]
Thanks @luckedea I'll give that a shot.

**Reply by edrock200** [reply]
> > Just in general. I'd say out of every ~100 1 or 2 will pause.
> 
> This could be extension-related, not necessarily visible in logs. It would help if you could repeat it without extensions.
> 
> Too many things can be happening here, this one is difficult because I can't replicate it.

[example-file.log](https://github.com/user-attachments/files/16315268/example-file.log)

Here's an example from today. You can see I unpaused it at 13:34:52. It always seems to resolve around par2 checking/downloading. 

I have now disabled plugins though and see if it continues to happen.

**Reply by luckedea** [reply]
Interesting, thank you. Par2 library is the subject of review & upgrade soon anyway.
